src/init.c

```diff
@@ -414,13 +448,26 @@ int read_cards(char *suggestion)
 				/* Get first count string */
```

Comment:
Yes, that is a good suggestion. (See also my comment about AVAILABLE DESIGN.)

Global variables are always initialized to 0, no need to do anything more there.

The problem is that after you change this, the nets won't load because of incompatible sizes. If it were not for this, I agree that we could wait with extra training until the code is final. Then I think we should run it one last time.